pages/chaoss.py

```diff
@@ -0,0 +1,127 @@
+from turtle import title
+from dash import html, callback_context, callback, dcc
+import plotly.express as px
+from dash.dependencies import Input, Output
+import plotly.express as px
+import dash_bootstrap_components as dbc
+from app import app
+import pandas as pd
+import datetime as dt
+from .visualizations import commits_activity as ca
+
+layout = dbc.Container([
+    
+    dbc.Row([
+        dbc.Col([
+            html.H1(children="Chaoss WIP Page - live update")
+            ]
+        )
+        ]
+    ),
+
+    dbc.Row([
+        dbc.Col([
+
+            dcc.Graph(id='commits-over-time1'),
+            html.Label(['Date Interval'],style={'font-weight': 'bold'}),
+            dcc.RadioItems(
+                id='time_interval',
+                options=[
+                         {'label': 'Day', 'value': 86400000},
+                         {'label': 'Week', 'value': 604800000},
+                         {'label': 'Month', 'value': 'M1'},
+                         {'label': 'Year', 'value': 'M12'}
+                ],
+                value='M1',
+                style={"width": "50%"}
+                ),
+            ],
+        ),
+
+        dbc.Col([
+            dcc.Graph(id='first-time-contributors'),
+            ],
+        ),
+
+        ]
+    ),
+
+], fluid= True)
+
+#call back for commits over time graph   
+@callback(
+    Output('commits-over-time1', 'figure'),
+    [Input('commits-data', 'data'),
+    Input('time_interval','value')]
+)
+def create_graph(data,interval):
+    df_commits = pd.DataFrame(data)
+
+    # reset index to be ready for plotly
+    df_commits = df_commits.reset_index()
+
+    #helper values for building graph 
+    today = dt.date.today()
+    x_r = []
+    x_name = "Year"
+    hover = "Year: %{x|%Y}"
+
+    #graph input values based on date interval selection
+    if interval == 86400000:
```

Comment:
These hard-coded numbers out to be written in the form:
DAY_INTERVAL = 86400000
etc.
at the top of the function to increase legibility.